src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenInParametersTests.cs

```diff
@@ -1165,6 +1182,21 @@ public void M1(in int arg2, in int arg3, in int arg4)
 42
 3
 3");
+
+            comp = CreateCompilationWithMscorlib46(text, new[] { ValueTupleRef, SystemRuntimeFacadeRef }, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular.WithPEVerifyCompatFeature());
+            CompileAndVerify(comp, verify: false, expectedOutput: @"
+3
+42
+2
```

Comment:
I'm also concerned about this case. Is there a way to describe the cases which have observable changes and exclude them from peverify-compat?